dds/DCPS/Service_Participant.cpp

```diff
@@ -459,11 +459,15 @@ Service_Participant::parse_args(int &argc, ACE_TCHAR *argv[])
       arg_shifter.consume_arg();
       got_info = true;
 
-    } else if ((currentArg = arg_shifter.get_the_parameter(ACE_TEXT("-DCPSRTISerialization"))) != 0) {
-      Serializer::set_use_rti_serialization(ACE_OS::atoi(currentArg));
+    }
```

Comment:
This line and the next are modified, but only the spacing and placement of `}` changed, let's try to avoid that.